docs/edgeql/ddl/links.rst

```diff
@@ -331,7 +331,7 @@ The following subcommands are allowed in the ``ALTER LINK`` block:
     Remove an :ref:`index <ref_datamodel_indexes>` defined on *index-expr*
     from this link.  See :eql:stmt:`DROP INDEX` for details.
 
-:eql:synopsis:`DROP default`
+:eql:synopsis:`RESET default`
     Remove the default value from this link.
```

Comment:
>  And if the latter, do we have/want a way to say "this property has no default, even though it does in the parent"?

Using empty set works, but in general it might be worth keeping `DROP` for this purpose.  Perhaps as a followup.